src/utils/validators/graphqlQueryValidators.ts

```diff
@@ -75,22 +78,42 @@ export const createDonationQueryValidator = Joi.object({
   amount: Joi.number()?.greater(0).required(),
   transactionId: Joi.when('safeTransactionId', {
     is: null || undefined || '',
```

Comment:
The value of `is` property is wrong or does not satisfy the intention of the original writer. The value will always evaluated to ''.